server/.env.environment

```diff
@@ -0,0 +1,10 @@
+# Database
```

Comment:
Remove envs as .env.example already pushed and also it has your private pandasai_api_key

Reply:
Removed the env file